tensorflow/core/ops/array_ops.cc

```diff
@@ -410,6 +410,27 @@ axis: Dimension along which to unpack.  Negative values wrap around, so the
 output: The list of tensors unpacked from `value`.
 )doc");
 
+REGISTER_OP("UnravelIndex")
+    .Input("indices: int32")
+    .Input("dims: int32")
+    .Output("output: int32")
+    .SetShapeFn([](InferenceContext* c) {
+      return Status::OK();
+    })
+    .Doc(R"doc(
+Converts a flat index or array of flat indices into a tuple of coordinate
+arrays.
+
+This is equivalent to numpy.unravel_index
```

Comment:
Can you put this into a `@compatility(numpy)` block (see other docstrings)?

Reply:
Thanks @martinwicke, the doctoring has been updated.